.vscode/debug.R

```diff
@@ -1,4 +1,5 @@
 env <- Sys.getenv()
 names <- names(env)
-rnames <- names[startsWith(names, "R_")]
+rnames <- names[startsWith(names, "LD_LIBRARY")]
```

Comment:
Is there a reason to use `startsWith(names, "LD_LIBRARY")` instead of exact `LD_LIBRARY_PATH`?

Reply:
Probably not. There is sometimes a "LIBRARY_PATH" environment variable defined but I believe this is only used at compilation (linker) time.

Reply:
Would you like to use exact `LD_LIBRARY_PATH` here to make it clear?

Reply:
I'll fix and resubmit